app/routes/about.jsx

```diff
@@ -68,15 +68,16 @@ export default function About() {
               1 million audio, video, film, and digital assets dating back to
               1947.
             </p>
-
-            <hr className="spaced-hr" />
-
-            <h2 className="static-link">
-              American Archive of Public Broadcasting
-            </h2>
-
-            <p className="static-section">
-              <img className="half-image right" src="/aapb.png" />
+            <hr className='spaced-hr' />
+            <h2>American Archive of Public Broadcasting</h2>
+            <h4>
+              {' '}
```

Comment:
typo?

Reply:
The icon looked weird when it was adjacent to the text. I'm sure there's a better CSS way to do it.